client/src/components/LoggedIn.jsx

```diff
@@ -40,7 +40,7 @@ class LoggedIn extends React.Component {
           <label htmlFor="zipCode">Zip Code:</label>
           <input type="number" inputMode="numeric" name="zipCode" value={this.state.zipCode} onChange={this.handleChange} />
 
-          <Link to="/find-restaurant" onClick={this.saveHost}>
+          <Link to="/find-restaurant" >
```

Comment:
Not sure why this is removed here. Is this part of the goal of the PR or something else? Just confused

Reply:
This has been removed right now bc it is overriding the host object made at login